examples/pytorch/dimenet/qm9.py

```diff
@@ -0,0 +1,218 @@
+"""QM9 dataset for graph property prediction (regression)."""
```

Comment:
I guess you can remove this file and use the built-in QM9 dataset instead?

Reply:
I add original graph and line graph loader and saver in this file, while built-in QM9 doesn't.

Reply:
Is it possible to inherit the built-in QM9 dataset and add code for additional pre-processing rather than doing everything from scratch?